src/firefly/test/edu/caltech/ipac/util/cache/CachePeerProviderFactoryTest.java

```diff
@@ -39,7 +39,7 @@
      * intended for connection test.  Not meant to be used in production.
      * change peerDiscovery=PubSub to peerDiscovery=MultiCast to test using multicast
      *
-     * Change log4j-test.properties to see LOG output
+     * Change to DEBUG level to see output.  i.e. Logger.setProLog(Level.DEBUG, null);
```

Comment:
The idea behind controlling the level in a property file outside the code was to be able to run it with different level without needing to compile. Is that not possible anymore?

Reply:
Yes, the idea behind controlling logs via property file is very useful and it's still supported by Firefly when running as a service.  This is where I think it shrines, because you have compiled code running and you cannot stop, restarted, or recompile it but yet still want to see more detailed logs.

But for test, which is a single run from start to finish then exit, logging is primary for developer's debugging.  In this scenario, it's just as easy to edit the test by setting logger's level, then rerun.  This is clean and I don't think it take away much of the original functionality.  Let me know if this is a good enough alternative.  If not, we can talk about it and come up with an acceptable solution.
To set log level, from anywhere in your code..
`Logger.setProLog(Level.DEBUG, null);`